src/base/static/client/mapseed-api-client.js

```diff
@@ -1,178 +1,17 @@
-      const layer = layers.find(layer => layer.id === collectionId);
-      const includePrivate = hasAdminAbilities(collectionId);
-      // if we are fetching a dataset id to be used in the dashboard,
-      // and the user has access to that dataset:
-      collection.fetchAllPages({
-        remove: false,
-        // Check for a valid location type before adding it to the collection
-        validate: true,
-        data: includePrivate
-          ? {
-              ...placeParams,
-              include_private_places: true,
-              include_private_fields: true,
-            }
-          : placeParams,
-        // get the dataset slug and id from the array of map layers
-        attributesToAdd: {
-          datasetSlug: layer.slug,
-          datasetId: layer.id,
-        },
-        attribute: "properties",
-        xhrFields: { withCredentials: includePrivate },
-
-        // Only do this for the first page...
-        pageSuccess: _.once(function(collection, data) {
-          pageSize = data.features.length;
-          totalPages = totalPages += Math.ceil(data.metadata.length / pageSize);
-
-          if (data.metadata.next) {
-            $progressContainer.show();
-          }
-        }),
-
-        // Do this for every page...
-        pageComplete: function() {
-          var percent;
-
-          pagesComplete++;
-          percent = (pagesComplete / totalPages) * 100;
-          $currentProgress.width(percent + "%");
-
-          if (pagesComplete === totalPages) {
-            _.delay(function() {
-              $progressContainer.hide();
-            }, 2000);
-          }
-        },
-
-        success: function(fetchedCollection) {
-          resolve(fetchedCollection, collectionId);
-        },
-
-        error: function(collection, err) {
-          layer.is_visible_default && setLayerError(collectionId);
-          // eslint-disable-next-line no-console
-          console.error(
-            `error loading place collection: ${collectionId}: err:`,
-            err,
-          );
-          reject(err);
-          // TODO: layer loading event; fix in layer UI PR
-        },
-      });
-    });
-    placeCollectionPromises.push(placeCollectionPromise);
-  });
-  return await Promise.all(placeCollectionPromises);
-};
-
-const getActivity = activityCollections => {
-  activityCollections.forEach(([activityCollection, successCallback]) => {
-    activityCollection.fetch({
-      success: successCallback,
-      error: (collection, error) => {
-        // eslint-disable-next-line no-console
-        console.error("Error fetching activity collection", error);
-      },
-    });
-  });
-};
+import activityClient from "./activity";
+import attachmentsClient from "./attachments";
+import commentsClient from "./comments";
+import datasetsClient from "./datasets";
+import placeClient from "./place";
+import placeTagsClient from "./place-tags";
+import supportClient from "./support";
```

Comment:
nit: I would suggest putting all of these under a single folder, like `mapseed-api-client/support.js`, so it doesn't look like we have a separate client for each entity.